gradlew.bat

```diff
@@ -0,0 +1,103 @@
+@rem
```

Comment:
Strictly speaking we could delete this file.  It occurs to me that I think I never have in similar situations, so definitely not a deal breaker.